text/0000-optimize-by-default.md

```diff
@@ -0,0 +1,166 @@
+- Start Date: 2015-03-10
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Make rustc and cargo produce optimized binaries by default.
+
+# Motivation
+
+
+Optimizing by default is a safety & usability feature that everyone benefits
```

Comment:
I'd be happy with no change after this discussion although that won't avoid confusion for future readers if this lands. I guess a parenthetical like "safety (in a general sense) & usability", or, e.g.

> Optimizing by default is a usability feature that everyone benefits from by protecting against certain common failure modes, but it's extra important for newcomers.